benches/benches/benchmarks/util.rs

```diff
@@ -454,22 +458,35 @@ pub fn create_2out_transaction(
         .build();
 
     let privkey: Privkey = PRIVKEY.into();
+    let witness: WitnessArgs = WitnessArgs::new_builder()
+        .lock(Bytes::from(vec![0u8; 65]).pack())
+        .build();
+    let witness_len: u64 = witness.as_bytes().len() as u64;
+    let non_sig_witnesses = vec![Bytes::new(); inputs_count - 1];
 
     let mut blake2b = ckb_hash::new_blake2b();
     let mut message = [0u8; 32];
     blake2b.update(&raw.hash().raw_data()[..]);
+    blake2b.update(&witness_len.to_le_bytes());
+    blake2b.update(&witness.as_bytes());
+    for w in &non_sig_witnesses {
+        let len: u64 = w.len() as u64;
+        blake2b.update(&len.to_le_bytes());
+        blake2b.update(&w);
+    }
     blake2b.finalize(&mut message);
     let message = H256::from(message);
-    let witness: Bytes = privkey
+    let sig: Bytes = privkey
         .sign_recoverable(&message)
         .expect("sign tx")
         .serialize()
         .into();
+    let witness = witness.as_builder().lock(sig.pack()).build();
 
-    raw.as_advanced_builder()
-        .witness(witness.pack())
-        .witness(witness.pack())
-        .build()
+    let mut witnesses = vec![witness.as_bytes().pack()];
+    witnesses.extend(non_sig_witnesses.into_iter().map(|w| w.pack()));
```

Comment:
Only the first witness is signed, the rest is not needed?

Reply:
For a verify group, we only need one signature.

Reply:
Thanks, I think it is not mandatory.

Reply:
It is mandatory unless you sign and fill the witnesses backward.